docs/Users_Guide/glossary.rst

```diff
@@ -6753,6 +6753,11 @@ METplus Configuration Glossary
 
      | *Used by:* PointStat
 
+   POINT_STAT_OUTPUT_FLAG_ORANK
```

Comment:
I realize that order doesn't matter to the code, but it matter to the user. I'd recommend moving the ORANK settings up in the list of options. I'd recommend listing it immediately after the ECNT entries... since the ECNT, ORANK, and RPS output are only compute for HiRA algorithm in Point-Stat. So it's useful to group them. That's why I listed them in that order [here](https://github.com/dtcenter/MET/blob/3a2405d6097a588a75993aae7e70917c326f04df/met/data/config/PointStatConfig_default#L283).

Reply:
The glossary is sorted alphabetically automatically so the order of items listed in this file won't affect the order. I can change the order in the wrappers.rst file so that it comes after ECNT.

Reply:
Commit 35846c4f reordered these values in wrappers.rst